MyWeatherApp/Extensions/LoadingLayer.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  LoadingLayer.swift
+//  MyWeatherApp
+//
+//  Created by abramovanto on 15.05.2024.
+//
+
+import SwiftUI
+
+struct LoadingLayer: ViewModifier {
+    var isLoading : Bool
+    var backgroundColor: Color
+    func body(content: Content) -> some View {
```

Comment:
Нужна пустая строка между функцией и проперти
```suggestion
    var backgroundColor: Color

    func body(content: Content) -> some View {
```